### DISCUSSION TITLE
Wildcards in the qodana.yaml file

### DISCUSSION THREAD

**Discussion by ahus1** [JetBrains/Qodana#106] [Q&A] [answered]
I tried to use wildcards for excludes (like `*.txt`) in the `qodana.yaml` file, and they don't seem to work. 

It would be good if that would be part of the docs (https://www.jetbrains.com/help/qodana/qodana-yaml.html) if this should be the case. If there is support for wildcards and I've just used it in a wrong way, I'd be happy to see a working example in the docs. 

Thanks!

**Comment by tiulpin** [comment, marked as answer]
It's not supported ATM, but will be – https://youtrack.jetbrains.com/issue/QD-1087/Wildcards-in-Exclusion-Paths
